ci-operator/config/openshift/release/openshift-release-master__cnv-nightly-4.18.yaml

```diff
@@ -32,13 +32,13 @@ releases:
       architecture: multi
       product: ocp
       relative: 1
-      stream: nightly
+      stream: 4-stable
```

Comment:
Hi @dollierp , what release would you really test here from the ones available at https://multi.ocp.releases.ci.openshift.org/?

Reply:
Hi,

For 4.18, we want to use the latest one from the `4-stable-multi` stream, currently 4.18.4.

For 4.19 we want to use the latest one from the `4-dev-preview-multi` stream, currently 4.19.0-ec.3.

Reply:
can you try these?

sprintly releases:
```
    prerelease:
      architecture: multi
      product: ocp
      version_bounds:
        lower: 4.19.0-0
        stream: 4-dev-preview
        upper: 4.20.0-0
```

stable releases:
```
  target:
    release:
      architecture: multi
      channel: candidate|fast|stable
      version: "4.18"
```

Reply:
Looks like it did the trick. At least the bot doesn't complain anymore :)